windowsforms/Sudoku/VB/sudoku/core/Game.vb

```diff
@@ -0,0 +1,57 @@
+Public Class Game
+
+    Public Event ShowClues(grid()() As Integer)
+    Public Event ShowSolution(grid()() As Integer)
+
+    Private ReadOnly HRow(8) As List(Of Integer)
+    Private ReadOnly VRow(8) As List(Of Integer)
+    Private ReadOnly ThreeSquare(8) As List(Of Integer)
+    Private ReadOnly grid(8)() As Integer
+
+    Private r As Random
```

Comment:
Inconsistent naming convention here. I'd use the `dotnet/runtime` coding style and make all of these `_camelCase`.

Reply:
I have not change the code from the archive I can easily change it using a CodeFix.

Reply:
@paul1956, I think it would be better to change it so it's consistent and follows the coding style from `dotnet/runtime` (previously, `dotnet/corefx`).

Reply:
The repository .EditorConfig only defines a coding style for C# and not VB except for CharSet. I think that should be fixed there then I can just run a Code Cleanup. Otherwise every VB project needs their own file,